libs/community/langchain_community/vectorstores/vdms.py

```diff
@@ -76,6 +76,41 @@ def _len_check_if_sized(x: Any, y: Any, x_name: str, y_name: str) -> None:
     return
 
 
+def _results_to_docs(results: Any) -> List[Document]:
+    return [doc for doc, _ in _results_to_docs_and_scores(results)]
+
+
+def _results_to_docs_and_scores(results: Any) -> List[Tuple[Document, float]]:
+    final_res: List[Any] = []
+    try:
+        responses, blobs = results[0]
+        if (
+            len(responses) > 0
+            and "FindDescriptor" in responses[0]
+            and "entities" in responses[0]["FindDescriptor"]
+        ):
+            result_entities = responses[0]["FindDescriptor"]["entities"]
+            # result_blobs = blobs
+            for ent in result_entities:
+                distance = round(ent["_distance"], 10)
+                txt_contents = ent["content"]
+                for p in INVALID_DOC_METADATA_KEYS:
+                    if p in ent:
+                        del ent[p]
+                props = {
+                    mkey: mval
+                    for mkey, mval in ent.items()
+                    if mval not in INVALID_METADATA_VALUE
+                }
+
+                final_res.append(
+                    (Document(page_content=txt_contents, metadata=props), distance)
+                )
+    except Exception as e:
+        print(f"Error while parsing results: {e}")  # noqa: T201
```

Comment:
Could you revert the print statement? If you need to log something feel free to use logger.error or raise an exception to the user

Reply:
Reverted to logger